vlsi/Makefile

```diff
@@ -64,7 +64,7 @@ else
 	VLSI_BB = $(sim_top_blackboxes)
 endif
 
-.PHONY: default verilog
+.PHONY: default verilog all
```

Comment:
Should this be here?

Reply:
No, also this would collide with #1324.

Reply:
pre-commit will block committing without this so I just added it in. I'll remove it.